### DISCUSSION TITLE
Unable to do screen share with conference using Android SDK

### DISCUSSION THREAD

**Discussion by rushikakpatel1989** [ant-media/Ant-Media-Server#6561] [Q&A] [answered]
Hi Team,

We are using Android SDK for developing our app. 
We are creating conferencing solution, It is not allowing us to screen share with conference.
We wanted to switch from camera to screen share but that is not working. We are using function changeVideoSource to change video, it is removing camera video but not showing screen to other connected users. It is just showing it on localVideo. 

Can you please share possible methods to achive this?

**Comment by lastpeony** [comment]
I created an issue
https://github.com/ant-media/Ant-Media-Server/issues/6580
and we will start working on it this week. It will be probably ready next monday in PR form. It might take another week for it to be available at maven.

**Comment by lastpeony** [comment, marked as answer]
This issue is solved and its merged to the master branch.
https://github.com/ant-media/WebRTC-Android-SDK/pull/98
We will release a new version to mvn this week and it will be included.

Feel free to re-open.
